Content.Client/EscapeMenu/UI/Tabs/KeyRebindTab.xaml.cs

```diff
@@ -74,6 +86,18 @@ void AddButton(BoundKeyFunction function)
                 _keyControls.Add(function, control);
             }
 
+            void AddCheckBox(string checkBoxName, bool currentState = false, Action<BaseButton.ButtonToggledEventArgs>? callBackOnClick = null)
```

Comment:
```suggestion
            void AddCheckBox(string checkBoxName, bool currentState, Action<BaseButton.ButtonToggledEventArgs>? callBackOnClick)
```
no need for default values